docs/transactions.pug

```diff
@@ -46,7 +46,36 @@ block content
     operation in a transaction, you need to pass the `session` as an option.
 
     ```javascript
-    [require:transactions.*basic example]
+    const Customer = mongoose.model('Customer', new Schema({ name: String }));
+
+    await Customer.createCollection();
+    const session = await mongoose.startSession();
+
+    // Start a transaction
+    session.startTransaction();
+
+    // This `create()` is part of the transaction because of the `session`
+    // option.
+    await Customer.create([{ name: 'Test' }], { session: session });
```

Comment:
I wonder if async await examples should use `exec()` for better stack traces?

Reply:
I agree that it's better to use [`Query#exec()`](https://mongoosejs.com/docs/api/query.html#query_Query-exec) in examples. Although for this it isn't necessary: `create()` returns a promise, and doesn't have an `exec()` function.